src/main/resources/pdiresources/jobs/control-center.kjb

```diff
@@ -395,6 +395,43 @@
       <yloc>64</yloc>
       <attributes_kjc/>
     </entry>
+    <entry>
+      <name>encounter_job</name>
+      <description/>
+      <type>JOB</type>
```

Comment:
i think its best in otherTransformations job, adding it there is similar to adding it to control center!

Reply:
I dont your  suggestion, do you mean  add  the job in other transformations?

Reply:
done, I have already added this  transformation  to  otherTransformations job